### DISCUSSION TITLE
Adaptive mode feature to ignore status bar icons.

### DISCUSSION THREAD

**Discussion by Xisrr1** [tribalfs/GalaxyMaxHzPub#165] [Ideas]
The data/wifi upload/download arrows always change and activates 120hz, will be nice if the app will ignore it and keep the 60hz. @tribalfs

**Comment by tribalfs** [comment]
Unfortunately it's impossible to isolate this from the statusbar/notification panel events.

**Reply by N-egaran** [reply]
I have this problem too, is there any solution to this problem?  😔
